src/injected-content/scss/_theatermode.scss

```
@@ -110,4 +110,12 @@ b-channel-page-wrapper .stage:not(.aspect-16-9).me-video-stage{
     position: relative !important;
 }
 
+/* Co-Stream Theatermode top-bar fix*/
+.costream-rollout {
+	top: 59px !important;
+}
+
+.costream-rollout::before {
```

Same as above, we need to scope this to theaterMode only.

```suggestion
.stage.theaterMode .costream-rollout::before {
```